Tavisca.Bootcamp.LanguageBasics.Exercise1/FixMultiplication.cs

```diff
@@ -0,0 +1,270 @@
+                n1x = n3 / n2;
+                string t1 = n1x.ToString();
+
+                for (int p = 0; p < t1.Length; p++)
+                {
+                    if (num1.ElementAt(p) == '?')
+                    {
+                        //Console.WriteLine("RES : " + t1.ElementAt(p));
+                        char rc = t1.ElementAt(p);
+                        int res = rc - '0';
+                        return res;
+                    }
+                }
+            }
+            else
+            {
+                //Console.WriteLine("RES : " + (-1));
+                return -1;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        public static int NumberTwoIncomplete(int indexStar, int indexEqual, int indexQuestionMark, string equation)
+        {
+            int c1 = 0;
+            int i = 0;
+            while (equation.ElementAt(i++) != '*')
+            {
+                c1++;
+            }
+            string nc1 = equation.Substring(0, c1);
+            int n1 = Int32.Parse(nc1);
+            //Console.WriteLine("nc1 : " + nc1);
+
+            // Number 2
+            int c2 = 0;
+            i = c1 + 2;
+            while (equation.ElementAt(i++) != '=')
+            {
+                c2++;
+            }
+            string nc2 = equation.Substring(c1 + 1, c2 + 1);
+            //int n2 = Int32.Parse(nc2);
+
+            //Console.WriteLine("nc2 : " + nc2);
+
+            // Number 3
+            int c3 = 0;
+            i = c1 + c2 + 3;
+
+            while (i < equation.Length - 1)
+            {
+                c3++;
+                i++;
+            }
+
+            string nc3 = equation.Substring(c1 + c2 + 3, c3 + 1);
+            //Console.WriteLine("nc3 : " + nc3);
+            int n3 = Int32.Parse(nc3);
+
+            int n2x = 0;
+            if (n3 % n1 == 0)
+            {
+                n2x = n3 / n1;
+                string t2 = n2x.ToString();
+
+                if (nc2.Length != t2.Length)
+                {
+                    //Console.WriteLine("RES : " + (-1));
+                    return -1;
+                }
+
+                for (int p = 0; p < t2.Length; p++)
+                {
+                    if (nc2.ElementAt(p) == '?')
+                    {
+                        //Console.WriteLine("RES : " + t2.ElementAt(p));
+                        char rc = t2.ElementAt(p);
+                        int res = rc - '0';
+                        return res;
+                    }
+                }
+            }
+            else
+            {
+                //Console.WriteLine("RES : " + (-1));
+                return -1;
+            }
+
+            throw new NotImplementedException();
+        }
+
+        public static int NumberThreeIncomplete(int indexStar, int indexEqual, int indexQuestionMark, string equation)
+        {
+            int c1 = 0;
+            int i = 0;
+            while (equation.ElementAt(i++) != '*')
+            {
```

Comment:
This code is repeated thrice in this program. Create a common method and use it.